inginious/frontend/app.py

```diff
@@ -307,6 +307,7 @@ def flask_internalerror(e):
     flask_app.privacy_page = config.get("privacy_page", None)
     flask_app.static_directory = config.get("static_directory", "./static")
     flask_app.webdav_host = config.get("webdav_host", None)
+    flask_app.jwt_key = config.get("jwt_key", None)
```

Comment:
This does not match the location you used in the `configuration.deploy.yaml` file.